index.html

```diff
@@ -16,48 +18,67 @@
 
 <body>
 
- <!-- navigation code here -->
+  <!-- navigation code here -->
+  <div class="navigation">
+    <h1>SpaceWalkers</h1>
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </nav>
+  </div>
 
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
+      <!-- image code here -->
+      <img src="img/main-header.png" alt="main header">
```

Comment:
is there supposed to be a space between the nav & top img?

Reply:
I adjusted the rem height of my navigation. This makes it look  like the model page.